src/MediaAccessibility/MediaAccessibility.cs

```diff
@@ -314,4 +314,34 @@ static partial class MAAudibleMedia {
 			return result;
 		}
 	}
+
+#if NET
+	[SupportedOSPlatform ("ios16.4")]
+	[SupportedOSPlatform ("maccatalyst16.4")]
+	[SupportedOSPlatform ("macos13.3")]
+	[SupportedOSPlatform ("tvos16.4")]
+#endif
+	public static partial class MAVideoAccommodations {
+#if NET
+		[SupportedOSPlatform ("ios16.4")]
+		[SupportedOSPlatform ("maccatalyst16.4")]
+		[SupportedOSPlatform ("macos13.3")]
+		[SupportedOSPlatform ("tvos16.4")]
+#else
+		[Mac (13, 3), TV (16, 4), iOS (16, 4)]
+#endif
+		[DllImport (Constants.MediaAccessibilityLibrary)]
+		[return: MarshalAs (UnmanagedType.I1)]
+		static extern bool MADimFlashingLightsEnabled ();
```

Comment:
Marshalling is not allowed in P/Invokes anymore :)

```suggestion
		static extern byte MADimFlashingLightsEnabled ();
```

Reply:
Ah right! Good catch!